### DISCUSSION TITLE
One Page Website Menu (Section Menu)

### DISCUSSION THREAD

**Discussion by aybarschill** [nolimits4web/swiper#6989] [Q&A]
Hello, I have installed the Swiper library into my HTML-based website. I've coded each page to be a slide, but I want to add a menu. I'd like it so that when a section in the menu is clicked, it scrolls to that corresponding slide. How can I achieve this?

**Comment by kaba4o4ekk** [comment]
hi. idk


пн, 28 авг. 2023 г. в 17:09, yunusemread ***@***.***>:

**Comment by Arhell** [comment]
> Hello, I have installed the Swiper library into my HTML-based website. I've coded each page to be a slide, but I want to add a menu. I'd like it so that when a section in the menu is clicked, it scrolls to that corresponding slide. How can I achieve this?

Hi, try
https://swiperjs.com/swiper-api#method-swiper-slideTo
